### PR TITLE
[Face] Adjust Face examples to fit the new testable design.

### PR DESCRIPTION
**Pull request by yangyuan**
Please use `[API Name] Brief description` as title.

## API Name (Kind)
`Face`

## Purpose
Refactor Face examples so that they can be cached by the new test runner.

### CODE REVIEW

File: samples/Face/Microsoft.Azure.CognitiveServices.Samples.Face.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netstandard1.4</TargetFrameworks>
```
Comment (yangyuan) [inline samples/Face/Microsoft.Azure.CognitiveServices.Samples.Face.csproj]: Add `netstandard1.4` to ensure compatibility, this is the the target the tester uses.

File: samples/Face/Microsoft.Azure.CognitiveServices.Samples.Face.sln [hunk 2]
```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27703.2042
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VerifyFaceToFace", "VerifyFaceToFace\VerifyFaceToFace.csproj", "{9090CBD6-25BE-4C58-A747-814D2E566956}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Azure.CognitiveServices.Samples.Face", "Microsoft.Azure.CognitiveServices.Samples.Face.csproj", "{9090CBD6-25BE-4C58-A747-814D2E566956}"
```
Comment (yangyuan) [inline samples/Face/Microsoft.Azure.CognitiveServices.Samples.Face.sln]: Change the name so that the tester can locate the samples.

File: samples/Face/Program.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Samples.Face
+{
+    using System;
+
+    class Program
+    {
+        static void Main(string[] args)
```
Comment (yangyuan) [inline samples/Face/Program.cs]: Main method wont be called by tester, do what ever you want in `main`.

File: samples/Face/VerifyFaceToFace.cs [hunk 4]
```diff
@@ -1,24 +1,23 @@
...
+    using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+
+    public static class VerifyFaceToFace
     {
-        public static void Main(string[] args)
+        public static async Task Run(string endpoint, string key)
```
Comment (yangyuan) [inline samples/Face/VerifyFaceToFace.cs]: Tester will automatically fill these two parameters.

File: samples/Face/VerifyFaceToFace.cs [hunk 5]
```diff
@@ -1,24 +1,23 @@
...
-            string apiKey = "ENTER YOUR KEY HERE";
-            IFaceClient client = new FaceClient(new ApiKeyServiceClientCredentials(apiKey))
+            IFaceClient client = new FaceClient(new ApiKeyServiceClientCredentials(key))
             {
-                Endpoint = "ENTER YOUR ENDPOINT HERE"
+                Endpoint = endpoint
             };
 
             List<string> imageFileNames =
-                new List<string> { "Family1-Dad1.jpg", "Family1-Dad2.jpg", "Family1-Son1.jpg" };
```
Comment (lebronJ) [inline samples/Face/VerifyFaceToFace.cs]: [Nit] Unnecessary change.
